frame/merkle-mountain-range/src/tests.rs

```diff
@@ -410,40 +424,43 @@ fn should_generate_batch_proof_correctly() {
 	register_offchain_ext(&mut ext);
 	ext.execute_with(|| {
 		// when generate proofs for a batch of leaves
-		let (.., proof) = crate::Pallet::<Test>::generate_batch_proof(vec![0, 4, 5]).unwrap();
+		let (.., proof) = crate::Pallet::<Test>::generate_batch_proof(vec![1, 4, 5]).unwrap();
```

Comment:
I would rather modify `vec![0, 4, 5]` to `vec![1, 5, 6]` and then the expected `BatchProof` should remain the same as before. Same for the tests below.